tests/integration/awslambda/test_lambda_api.py

```diff
@@ -1301,13 +1309,15 @@ def test_notfound_and_invalid_routingconfigs(
 
 
 @pytest.mark.skipif(condition=is_old_provider(), reason="not supported")
-@pytest.mark.skip_snapshot_verify(
-    paths=[
-        # new Lambda feature: https://docs.aws.amazon.com/lambda/latest/dg/snapstart.html
-        "$..SnapStart"
-    ]
-)
 class TestLambdaRevisions:
+    @pytest.mark.skip_snapshot_verify(
+        # The RuntimeVersionArn is currently a hardcoded id and therefore does not reflect the ARN resource update
+        # from python3.9 to python3.8 in update_function_configuration_response_rev5.
+        paths=[
+            "update_function_configuration_response_rev5..RuntimeVersionConfig.RuntimeVersionArn",
+            "get_function_response_rev6..RuntimeVersionConfig.RuntimeVersionArn",
```

Comment:
```suggestion
            "$..RuntimeVersionConfig.RuntimeVersionArn",
```

or was there a particular reason for only including those?

Reply:
I still wanted the validation that the `RuntimeVersionArn` does not change between revisions and not ignore it completely for the entire test